api/HTMLInputElement.json

```diff
@@ -2323,7 +2323,10 @@
               "safari": {
                 "version_added": "16"
               },
-              "safari_ios": "mirror",
+              "safari_ios": {
+                "version_added": false,
+                "impl_url": "https://webkit.org/b/234009"
```

Comment:
This bug is marked as duplicate and it points to fixed bug. Do you have a better bug reference?

Reply:
Ah apologies yes I do will update that